src/bin/cargo-fmt.rs

```diff
@@ -288,16 +288,16 @@ fn format_files(files: &[PathBuf],
         println!("");
     }
     let mut command = try!(Command::new("rustfmt")
-        .stdout(stdout)
-        .args(files)
-        .args(fmt_args)
-        .spawn()
-        .map_err(|e| match e.kind() {
-            std::io::ErrorKind::NotFound => {
-                std::io::Error::new(std::io::ErrorKind::Other,
-                                    "Could not run rustfmt, please make sure it is in your PATH.")
-            }
-            _ => e,
-        }));
+                               .stdout(stdout)
```

Comment:
I tested with this piece of code, and it turns out that rustfmt cannot format this snippet (tested on 0.8.3() and the latest master (75a1386)). 
Specifically, the following code will be left as is.
```rust
    let mut command = try!(Command::new("rustfmt")
.stdout(stdout)
.args(files)
.args(fmt_args)
.spawn()
.map_err(|e| match e.kind() {
std::io::ErrorKind::NotFound => {
std::io::Error::new(std::io::ErrorKind::Other,
"Could not run rustfmt, please make sure it is in your PATH.")
}
_ => e,
}));
```